tf_quant_finance/examples/jupyter_notebooks/Swap_Curve_Fitting.ipynb

```diff
@@ -0,0 +1,533 @@
M/fMxYhWg1e/iQDBwoqpY5FLohl2wpkIrf+I9djMJrxzy9zca5VjyVzYuHtyZ1HiEh6/t4e+MMtcdBF+OI/G/KRebxa6kjkYli2rUDOrf/IxVgsFnz09TEUnm3EPTN1iAjykjoSEVE3N6UMz9wzDgODPPH2+jzknqyVOhK5EJZtK+ga2ebWf+Qqth08g9255Zh17UCM1QVIHYeI6AIeKgUeu2Ukgn3VeHNtLgpO1UkdiVwEy7YV8FAbciU5RTX4/PsTGBWpReqkCKnjEBFdklqlwB/mx8G/vzv+8UUOCs82SB2JXADLthUo5J1/rQYjR7bJuZVWN+Nf648gzF+De2cOh8idR4jIznl6KPH4/Dh4e6rw2ufZOFl2TupI5ORYtq1ArVIAAE+vIqfW1KrH61/kQKmQYcmcWLgpufMIETmGfho3PDE/Dhp3BV79LAsllU1SRyInxrJtBWqVHADQ2m6UOAmRdRhNZrz1ZR7qm/R4ZHYMfLxUUkciIrosPl4q/HFBPFRuMry8Kgul1c1SRyInxbJtBUqFDHKZiJY2jmyT87FYLPh423EcO9OAu6dHYXBIP6kjERFdEb/+7nhifjxkMgEvr8pCZV2r1JHICbFsW4naXY4WjmyTE9p+6Cx2ZpdhxvhwJEQHSh2HiOiqBPh44PH58TCZLXjx00xUN7RJHYmcDMu2lahVCs7ZJqeTV1yLVd8WIn6oH2ZPGSR1HCKiPhHip8bj8+OgN5jw0qeZqDvXLnUkciK9KtvFxcVIT09HUlIS0tPTcerUqQuuMZlMWLZsGRITEzF16lSsXr26+227d+/G7NmzER0djRdeeOGCx27ZsgWzZs1CSkoKZs2ahZqaGuWiNkYAACAASURBVADAG2+8gYSEBKSmpiI1NRXLli27wg/T9jxUcs7ZJqdSXtuCt9cdQYifGvemcOcRInIuAwI8sTQ9Di3tBry0KguNzR1SRyInIe/NRc8++ywWLlyI1NRUrF+/Hs888wxWrlx53jUbN25ESUkJtm3bhoaGBqSlpSEhIQGhoaEICwvD8uXLsXXrVuj1+vMel5ubizfffBMffvghtFotmpqaoFQqu9+elpaGJ598sg8+VNvSqBT8yZicRnObAf9YkwO5TMCSubFwd+vVlw4iIocSEeSFR+eNxKufZeOlVVn448J4eHkoe34g0W/ocWS7trYW+fn5SElJAQCkpKQgPz8fdXXnn7y0ZcsWzJs3D6IowsfHB4mJidi6dSsAIDw8HDqdDnL5hd+gV6xYgUWLFkGr1QIAPD094ebmdtUfmNQ8VJyzTc7BaDLj7XV5qG1sx0M3x8Cvn7vUkYiIrGZoaH8smRuL6oY2vLoqi1NC6ar1ODxVXl6OgIAAyGSde+jKZDL4+/ujvLwcPj4+510XHBzc/XpQUBAqKip6DFBUVITQ0FDceuutaG1txdSpU/HAAw9A+OlX1Js3b8bu3buh1WrxyCOPID4+/rI+QF9fzWVdfzW0Ws/ul/18PJBZWHPen5HzcKX7+t6GPBScrsfv0+Mx4ZowqePYlCvdZ1fHe+0aenuftVpPaDQq/OX9/XhjbS7+8rtr4fHTGRpk/+zt81ny3wWbTCYcO3YMH3zwAfR6Pe69914EBwcjLS0N8+fPx/333w+FQoE9e/bgwQcfxJYtW+Dt7d3r56+tbYbZbP1j07VaT1RX/7wpvmi2oK3DiPKKRshlXIfqTH59r51ZRV0rNuw8iSlxwRgZ4e0yHzfgWvfZ1fFeu4bLvc9hvu54IG0E3voyD0+/vQdLb4nj4V0OQKrPZ1EULjnA22MLDAoKQmVlJUwmE4DOclxVVYWgoKALrisrK+t+vby8HIGBPW8LFhwcjOTkZCiVSmg0Gtx4443IyckBAGi1WigUnT9JTpgwAUFBQSgsLOzxOe2B2r0zd2sHp5KQ41q36yTkcgFpk7jzCBG5nvihWtx30wicKG3E61/kQG8wSR2JHFCPZdvX1xc6nQ6bNm0CAGzatAk6ne68KSQAkJycjNWrV8NsNqOurg7bt29HUlJSjwFSUlKwe/duWCwWGAwG7Nu3D1FRUQCAysrK7usKCgpQWlqKiIiIy/oApeLBUyTJwZ2uaMKBgipMHR2GfmouECIi1zQmyh/3zNTh6Ol6/PPLPBiMZqkjkYPp1TSS5557Dk899RTeeusteHl5dW/ft3jxYixZsgQxMTFITU1FdnY2pk2bBgB46KGHEBbWOb8zIyMDS5cuRXNzMywWCzZv3ozly5dj0qRJmDlzJvLy8jBjxgyIooiJEydi7ty5AIBXX30VR44cgSiKUCgUePHFF7sXUtq7riPbubCCHNXanSehVskxfdwAqaMQEUnq2uggGIxmfLj1GP61Pg8PpEVziij1mmCxWKw/oVlCUs3ZLiptxPKPDuHReSMRO9jX6u+fbMcV5nceP9OA//vkMOZeNxgzxodLHUcSrnCfqRPvtWvoi/u8PeMM/ru9EGN1/rhv1giIIs8bsDf2OGdb8gWSzurnaSQc2SbHYrFYsGZHEfpplLhxVKjUcYiI7Ebi6DAYTGas/r4IcpmIRTN1POCLesSybSVdCyS51zY5mtyTtThxthG3TxsGNwVX3hMR/dL0ceEwGMxYt7sYSrmI25Miu7crJroYlm0r8XDjnG1yPGaLBV/sOAltfxUmjQzu+QFERC5o1oSB0BvN2LLvNORyEQtuHMrCTZfEsm0lcpkIlVKGljaObJPjOFhQhTNVzVg8azgX/xARXYIgCJgzZRD0RhO2Z5yFUi7DnCmDWLjpoli2rUitknPONjkMo8mML3edRIhWjXG6AKnjEBHZNUEQsODGoTD+NMKtlIu4aaJjbE9MtsWybUUeKgXnbJPD2J1bjqr6NjwyJ4Yr7ImIekEQBNyWFAmDsXMOt0IhYvo419zBiS6NZduK1Co552yTQ9AbTNiwuxiDQ7wQN8RP6jhERA5DFATcPUPXvUuJQiYicXSY1LHIjrBsW5FapUBFXavUM
```

Comment:
Can't find 'Building the Bloomberg Interest Rate Curve – Definitions and Methodology'. Could you add a more precise link?

Reply:
I could not find it in the public internet as well.

Reply:
This is fine then. Thanks!